wazo_phoned/plugin_helpers/client/http.py

```diff
@@ -0,0 +1,247 @@
+
+    def get(self, profile):
+        args = UserUUIDSchema().load(request.args)
+        xivo_user_uuid = args['xivo_user_uuid']
+
+        response_rendered = render_template(
+            self.template,
+            xivo_user_uuid=xivo_user_uuid,
+            xivo_proxy_url=_build_next_url('menu'),
+        )
+
+        return Response(response_rendered, content_type=self.content_type, status=200)
+
+
+class ClientInput(AuthResource):
+
+    content_type = None
+    template = None
+
+    def __init__(self, vendor, auth_client, dird_client):
+        self.vendor = vendor
+        self.auth_client = auth_client
+        self.dird_client = dird_client
+        super().__init__()
+
+    def get(self, profile):
+        args = UserUUIDSchema().load(request.args)
+        xivo_user_uuid = args['xivo_user_uuid']
+
+        response_rendered = render_template(
+            self.template,
+            xivo_user_uuid=xivo_user_uuid,
+            xivo_proxy_url=_build_next_url('input'),
+        )
+
+        return Response(response_rendered, content_type=self.content_type, status=200)
+
+
+class ClientLookup(AuthResource):
+
+    MAX_ITEM_PER_PAGE = None
+    content_type = None
+    template = None
+
+    def __init__(self, vendor, auth_client, dird_client):
+        self.vendor = vendor
+        self.auth_client = auth_client
+        self.dird_client = dird_client
+        super().__init__()
+
+    def get(self, profile):
+        args = LookupSchema().load(request.args)
+        limit = args['limit'] if args['limit'] is not None else self.MAX_ITEM_PER_PAGE
+        offset = args['offset']
+        term = args['term']
+        xivo_user_uuid = args['xivo_user_uuid']
+
+        user_tenant = self._get_user_tenant_uuid(xivo_user_uuid)
+
+        try:
+            results_lookup = self.dird_client.directories.lookup_user(
+                profile=profile,
+                user_uuid=xivo_user_uuid,
+                term=term,
+                tenant_uuid=user_tenant,
+            )
+        except HTTPError as e:
+            response = getattr(e, 'response', None)
+            status_code = getattr(response, 'status_code', None)
+            reason = getattr(response, 'reason', str(e))
+            return output_error(status_code, str(reason))
+        except ConnectionError:
+            raise WazoDirdConnectionError()
+
+        logger.debug('Raw lookup results: %s', results_lookup)
+        formatter = _PhoneResultFormatter(results_lookup)
+        formatted_results = formatter.format_results()
+        formatted_results.sort(key=attrgetter('name', 'number'))
+
+        if limit is not None:
+            paginated_results = formatted_results[offset : offset + limit]
+        else:
+            paginated_results = formatted_results[offset:]
+
+        total_results = len(formatted_results)
+
+        response_rendered = render_template(
+            self.template,
+            results=paginated_results,
+            xivo_proxy_url=request.base_url,
```

Comment:
Do you have a use case to have this value? On wazo-dird it will be to keep dird-phoned url
Maybe when there are a proxy on top of phoned?

Also there are no real reason to keep the name `xivo` in the template, we can rename it to wazo or just remove the prefix ...Same comment for xivo_user_uuid